JD2Resume.AI-main/.env

```
@@ -0,0 +1 @@
+VITE_GEMINI_API_KEY=AIzaSyCtyMt1DdTFxn4IEuvAKmiriqao3RbIiD8
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)